typescript/packages/wallets/viem/src/ViemEVMWalletClient.ts

```diff
@@ -68,6 +77,26 @@ export class ViemEVMWalletClient extends EVMWalletClient {
         return { signature };
     }
 
+    async signTransaction(transaction: EVMTransaction): Promise<{ signature: string }> {
```

Comment:
Hmm ideally:
1. We should add the signMessage method to the EVMWalletClient. All wallet clients should implement it (i think most do)
2. We can move this method to the EVMWalletClient and then all wallets will inherit this method :)

Up for doing that instead?

Reply:
I've realized that I probably don't actually need this method, so this will be a low priority task for me